context/src/main/java/io/grpc/Deadline.java

```diff
@@ -182,4 +179,42 @@ public long read() {
       return System.nanoTime();
     }
   }
+
+  private static <T> T checkNotNull(T reference, Object errorMessage) {
+    if (reference == null) {
+      throw new NullPointerException(String.valueOf(errorMessage));
+    }
+    return reference;
+  }
+
+  private static class LogExceptionRunnable implements Runnable {
```

Comment:
Ah, you could have just dropped it. It wasn't necessary to begin with (in c8648dc54). This works too.